src/Text/Pandoc/Writers/OpenDocument.hs

```diff
@@ -431,33 +447,37 @@ blockToOpenDocument o bs
       figure attr caption source title | null caption =
         withParagraphStyle o "Figure" [Para [Image attr caption (source,title)]]
                                   | otherwise    = do
+        let (ref, _, _) = attr
```

Comment:
A common idiom is to destructure Attr in the definition:

``` haskell
figure  attr@(ident, _, _) caption ...
```

This names both the whole Attr triple as well as the identifier.

Reply:
done.